packages/content-conformance/src/rules/ensure-valid-nav-paths.js

```diff
@@ -0,0 +1,71 @@
+import flat from 'flat'
+
+// Recursively traverse json nav data, and validate node paths against flat list of files from context
```

Comment:
I don't think this is recursive anymore 😄

Reply:
:hmm-yes:

Reply:
Fixed!